@ndlib/gatsby-theme-marble/src/components/Internal/CreateAccount/index.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import typy from 'typy'
+import MaterialButton from 'components/Internal/MaterialButton'
+import TextField from 'components/App/FormElements/TextField'
+import TextArea from 'components/App/FormElements/TextArea'
+import { createNewUser } from 'store/actions/loginActions'
+import style from 'components/App/FormElements/style.module.css'
+
+const CreateAccount = ({ loginReducer, dispatch }) => {
+  const claims = typy(loginReducer, 'token.claims').safeObject
+  const [name, changeName] = useState(claims.name)
+  const [email, changeEmail] = useState(claims.email)
+  const [bio, changeBio] = useState('')
+
+  return (
+    <form className={style.loginArea}>
+      <p>Welcome new user.</p>
+      <TextField
+        id='profileName'
+        label='Name'
+        defaultValue={name}
+        onChange={(event) => {
+          changeName(event.target.value)
+        }}
+      />
+      <TextField
+        id='profileUsername'
```

Comment:
Should this be profileUserName (in-line with the other changes from username -> userName?)

Reply:
Technically it doesn't matter here, but yes consistency is good so I'll change this.